unit.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"errors"
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/opencontainers/ocitools/config"
```

Comment:
Travis [doesn't like this import][1], and you don't seem to use `config.…` in `unit.go` anyway, so you can probably drop this line.

[1]: https://travis-ci.org/opencontainers/ocitools/builds/128853061#L193